### PR TITLE
Fixed ParlAI entry point

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
Fixed the entry point into parlai so that:
- Things don't catastrophically fail when you upgrade one of parlai's dependencies
- You can also run with `python -m parlai` as an equivalent to running the command. This is good because it enables @emilydinan to do `python -m pdb -m parlai` and get her debug functionality.

**_This requires everyone rerun `python setup.py develop`_**. Furthermore, we should probably release a new pypi release with this.

**Testing steps**
Manual testing.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,15 @@
     readme = f.read().split('--------------------')[-1]
 
 with open('requirements.txt') as f:
-    reqs = f.read()
+    reqs = []
+    for line in f:
+        line = line.strip()
+        if '>' in line:
+            reqs.append(line.split('>')[0])
+        elif '=' in line:
```
Comment (jxmsML) [inline setup.py:29]: we won't be expecting '~=' in , '<=' equirements.txt in the future?

Comment (jxmsML) [reply setup.py:29]: only this nit, other LGTM.

Comment (stephenroller) [reply setup.py:29]: Does ~= work?

Comment (stephenroller) [reply setup.py:29]: Maybe I can just use line.split('=').rstrip('><=~')...

Comment (jxmsML) [reply setup.py:29]: yeah that could work.

**Pull request merged**